### PR TITLE
Add boehmgc

### PR DESCRIPTION
**Pull request by asmeurer**
It is needed for inkscape. https://github.com/conda-forge/inkscape-feedstock/pull/7

Checklist

- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by asmeurer** [comment]
How do you add tests for this sort of thing? It's just a shared library.

**Comment by marcelotrevisani** [comment]
> How do you add tests for this sort of thing? It's just a shared library.

You can test if the shared object is present then, I will comment an example of that, just 1 min

File: recipes/boehmgc/build.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+cp $RECIPE_DIR/license.txt .
+
+./configure --prefix=$PREFIX
+make
```
Comment (marcelotrevisani) [inline recipes/boehmgc/build.sh]:
```suggestion
make -j${CPU_COUNT}
```

File: recipes/boehmgc/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  url: https://www.hboehm.info/gc/gc_source/gc-{{ version }}.tar.gz
+  sha256: 436a0ddc67b1ac0b0405b61a9675bca9e075c8156f4debd1d06f3a56c7cd289d
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+
```
Comment (marcelotrevisani) [inline recipes/boehmgc/meta.yaml]:
```suggestion
test:
  commands:
    - test -f ${PREFIX}/lib/libboehmgc${SHLIB_EXT}   # [unix]
    - if not exist %LIBRARY_LIB%\boehmgc.lib exit 1       # [win]
```

So you can test if the shared objects are available in the package, that is just an example, I don't know the name of those libs which this package generates

Comment (asmeurer) [reply recipes/boehmgc/meta.yaml]: OK. I think it's just called `gc`.

**Comment by isuruf** [comment]
Duplicate of `bdw-gc-feedstock`

**Comment by asmeurer** [comment]
@isuruf conda-forge needs to include more metadata in the feedstocks. I searched for "boehmgc" on the conda-forge GitHub and it turned up nothing.

**Pull request closed without merging**